doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+# High Fidelity Neutronics Model for Lead-cooled Fast Reactor (LFR)
+
+*Contact: Hansol Park, hansol.park@anl.gov*
+
+This VTB model provides a high fidelity neutronics model for a representative example of a lead-cooled fast reactor with an annular MOX (UPuO) fuel whose design is based on an early iteration of an LFR-prototype assembly provided by Westinghouse Electric Company, LLC [!citep](WECLFR). The original purpose of this model is to characterize the impact of various sources of uncertainties, such as theoretical and experimental uncertainties, instrumentation uncertainties, manufacturing tolerances, correlation uncertainties, and the method and simulation uncertainties, on peak cladding, fuel and coolant temperatures in the system. For this purpose, high fidelity neutronics (fine mesh heterogeneous transport - Griffin) and thermalhydraulics (computational fluid dynamics - NekRS) calculations were performed to compute hot channel factors (HCFs) [!citep](HCFReport) using a 3D heterogeneous single assembly steady-state problem. In this VTB model, only the neutronics standlone model is explained. 
+
+First, the LFR model is described, followed by the cross section generation procedure using MC2-3 [!citep](MCC3). Then, the Griffin standalone calculation setting with the discontinuous fininte element method (`DFEM`) discrete ordinate (`SN`) scheme with the Coarse Mesh Finite Difference (`CMFD`) acceleration is explained and the results are discussed.
+
+## LFR model description
+
+The key parameters of the LFR model are listed in [lfrkeyparam]. The power output is 950 MWth (~450 MWe) for the whole core and the nominal power of an inner core fuel assembly is 3.7 MWth. The fuel assembly pitch is 16.3 cm with a 4 mm lead-filled gap between assemblies and 3.5 cm thick duct wall as shown on the right in [lfrgeom]. Each assembly contains 127 cladded fuel pins arranged in a triangular lattice with pitch 1.33 cm within a hexagonal wrapper (duct). Each fuel pin has a cold fuel inner/outer diameter of 4.00/8.55 mm respectively, a fuel-cladding gap of 0.175 mm and a cladding outer diameter of 10.7 mm with a cladding thickness of 0.90 mm, depicted in [annularfuel]. The center hole and gap are filled with helium. To minimize the flow speed and consequently mitigate corrosion issues, a relatively wide lattice (P/D=1.24) design is adopted. Grid spacers are planned to maintain pin spacing, rather than the wire wrap used in conventional SFR designs.
```

Comment:
I think it is more accurate to differentiate between model and design. For example, it is better to say `the LFR design is described`.

Reply:
The head `LFR model description` is updated to `Description of LFR design`.